docs/en/pid-data-model.rst

```diff
@@ -0,0 +1,400 @@
+      - `[OID.IDA. Section 5.1] <https://openid.net/specs/openid-connect-4-identity-assurance-1_0-13.html#section-5.1>`_
+    * - **assurance_level**
+      - [NSD]. MUST be set to high
+      - `[OID.IDA. Section 5.1] <https://openid.net/specs/openid-connect-4-identity-assurance-1_0-13.html#section-5.1>`_
+    * - **evidence**
+      - [SD]. JSON Array. Each element is the electronic evidence of the user identification during the PID issuing phase. It MUST contain at least the following claims:
+
+            - **type**: MUST be set to ``electronic_record``
+            - **record**: JSON object (see the table below)
+      - `[OID.IDA. Section 5.1] <https://openid.net/specs/openid-connect-4-identity-assurance-1_0-13.html#section-5.1>`_
+
+
+The ``record`` MUST have at least the following sub parameters:
+
+.. list-table:: 
+  :widths: 20 60 20
+  :header-rows: 1
+
+  * - **Claim**
+    - **Description**
+    - **Reference**
+  * - **type** 
+    - identification of the national eID framework used by the User. For example ``eidas.it.cie`` means that the CIE id identification scheme is used by the User. 
+    - `[OID.IDA. Section 5.1.1.2] <https://openid.net/specs/openid-connect-4-identity-assurance-1_0-13.html#section-5.1.1.2>`_
+  * - **source**
+    - JSON Object cointaining the follwoing mandatory claims:
+
+      - **organization_name**: Name of the Organization handling the eID used by the User
+      - **organization_id**: Identification code for the Organization. It MUST be set to the *IPA Code* of the Organization
+      - **country_code**: String representing country in `[ISO3166-1] Alpha-2 (e.g., IT) or [ISO3166-3] syntax <https://www.iso.org/iso-3166-country-codes.html>`_.
+    - `[OID.IDA. Section 5.1.1.2] <https://openid.net/specs/openid-connect-4-identity-assurance-1_0-13.html#section-5.1.1.2>`_
+
+.. warning::
+  Note that the sub-claims of the **evidence** parameter are not selectively disclosable separately, thus, for example, the User cannot give only the *record type* without disclosure the *record source* (organization name, identifier and country that hendles the User identity proofing). 
+
+Finally, the ``claims`` parameter contains the following mandatory claims:
+
+.. list-table:: 
+    :widths: 20 60 20
+    :header-rows: 1
+
+    * - **Claim**
+      - **Description**
+      - **Reference**
+    * - **given_name**
+      - [SD]. Current First Name
+      - `[OpenID Connect Core 1.0, Section 5.1] <http://openid.net/specs/openid-connect-core-1_0.html>`_
+    * - **family_name**
+      - [SD]. Current Family Name
+      - `[OpenID Connect Core 1.0, Section 5.1] <http://openid.net/specs/openid-connect-core-1_0.html>`_
+    * - **birthdate**
+      - [SD]. Date of Birth
+      - `[OpenID Connect Core 1.0, Section 5.1] <http://openid.net/specs/openid-connect-core-1_0.html>`_
+    * - **place_of_birth**
+      - [SD]. Place of Birth. JSON Object with the following subclaims:
+
+        - **country**
+        - **locality**
```

Comment:
Not sure this will work

Please align with PID rulebook v1.0.0, it says

[ARF] specifies ‘Place of Birth’ as an optional data element in a PID attribute. This document  defines the following data elements related to this PID attribute: 
∙ birth_place (optional) 
∙ birth_country (optional) 
∙ birth_state (optional) 
∙ birth_city (optional) 

or with a compromise like the one below, that mixes the [italian oidc schema](https://italia.github.io/spid-cie-oidc-docs/en/attributi_utente.html) and the ARF

````
"place_of_birth":{
    "region":"MI",
    "locality":"F205",
    "country": ...,
    "state": ...,
    "city": ...,
}
````